contrib/mesos/pkg/scheduler/meta/annotations.go

```diff
@@ -25,7 +25,6 @@ const (
 	TaskIdKey                = "k8s.mesosphere.io/taskId"
 	SlaveIdKey               = "k8s.mesosphere.io/slaveId"
 	OfferIdKey               = "k8s.mesosphere.io/offerId"
-	ExecutorIdKey            = "k8s.mesosphere.io/executorId"
```

Comment:
We do need at least executor resource information in the task for procurement if no executor is set in the current offer.